charts/liv-chat/templates/deployment.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   labels:
     draft: {{ default "draft-app" .Values.draft }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+{{- if .Values.cluster.scService.podLabels }}
+{{ toYaml .Values.cluster.scService.podLabels | indent 8 }}
```

Comment:
`indent 8` is incorrect. This will break the yaml file. Check the number of spaces and update it accordingly.

Reply:
updated this

Reply:
updated this to `indent 4`